cpp/log/cert.cc

```diff
@@ -92,6 +93,24 @@ Cert::Status Cert::LoadFromDerString(const std::string& der_string) {
   return Cert::TRUE;
 }
 
+Cert::Status Cert::LoadFromDerBio(BIO *bio_in) {
+  if (x509_ != NULL) {
```

Comment:
Not to mention that this should just be `if (x509_)` anyway, neatly dodging the whole issue. :wink: